CODEOWNERS

```diff
@@ -1,5 +1,5 @@
 # Protocol owners are not generally differentiated by sub-area for simplicity
-/** @jordanschalm @AlexHentschel @durkmurder @zhangchiqing @peterargue @Kay-Zee @janezpodhostnik @tarakby
+/** @onflow/flow-core-protocol
```

Comment:
my comment is late and should have been added to the previous PR https://github.com/onflow/flow-go/pull/6851. By adding all the team's list, any PR on the repo will notify all the team members, even in areas they are not owning.
In my case at least, I prefer to keep ownership of the `crypto` subfolder only so I don't have to be notified to review the other PRs.

Reply:
By using a team rather than a direct list of usernames, we can avoid the notification overload (more details in the thread [here](https://flow-foundation.slack.com/archives/C071612SJJE/p1736210052911569)).